### PR TITLE
Switch to Acquia CMS

### PR DESCRIPTION
**Pull request by balsama**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,15 +10,18 @@
         }
     ],
     "require": {
+        "acquia/acquia_cms": "^1.1",
         "acquia/blt": "^12",
         "acquia/blt-phpcs": "^1.0",
-        "acquia/lightning": "^5",
+        "acquia/drupal-environment-detector": "^1",
+        "acquia/memcache-settings": "^1",
```
Comment (balsama) [inline composer.json:17]: These came from the [Acquia CMS Project](https://github.com/acquia/acquia-cms-project/blob/develop/composer.json). I'm not sure why the project is pulling them in and not `acquia/acquia_cms` directly. @msherron might know?

**Comment by balsama** [comment]
Some failures because ACMS pins Content Hub. See conversation here: https://acquia.slack.com/archives/C01D0CZGQSX/p1617112683021300

File: composer.json (hunk 1 above)
Comment (msherron) [reply composer.json:18]: @balsama - those to packages have scaffolding changes they need to make in settings.php, and they did not work as expected when I only relied on [acquia/acquia_cms to make the changes](https://github.com/acquia/acquia_cms/blob/develop/composer.json#L22) (despite having acquia_cms in the allowed-packages array in acquia-cms-project). Maybe I missed something, but that was the workaround.

**Comment by danepowell** [comment]
Per Slack, it seems like the CMS team is working to update that dependency. Ping me here when that's done and I'll review.

**Comment by balsama** [comment]
@danepowell this is ready for review. ACMS currently does not support the previous minor version of Drupal. So I've commented out the "[Integrated test on previous minor Drupal core version](78a35ae8cd6d5366ca5485R32)" test for now.

**Pull request merged**